File: app/src/main/java/ua/leonidius/rtlnotepad/MyApplication.java
package ua.leonidius.rtlnotepad;
import android.app.*;
import ru.elifantiev.android.roboerrorreporter.*;

public class MyApplication extends Application
{

	@Override
	public void onCreate()
	{
		RoboErrorReporter.bindReporter(this);
		super.onCreate();
	}

}

File: app/src/main/java/ua/leonidius/rtlnotepad/MyApplication.kt
package ua.leonidius.rtlnotepad

import android.app.Application
import com.chibatching.kotpref.Kotpref
import ru.elifantiev.android.roboerrorreporter.RoboErrorReporter

class MyApplication : Application() {

    override fun onCreate() {
        RoboErrorReporter.bindReporter(this)
        Kotpref.init(this)
        super.onCreate()
    }

}